### PR TITLE
edit readme to provide more details on cockroachdb

### PR DESCRIPTION
**Pull request by steeling**

### CODE REVIEW

**Comment by steeling** [comment]
Feel free to patch this branch, make edits yourself and resubmit if I'm not able to address feedback. Or if there is no feedback feel free to merge yourself

File: README.md [hunk 1]
```diff
@@ -76,23 +56,38 @@ When a USS wants to plan a flight, the “planning USS” performs the following
     of the world, industry provided OAuth solutions decided by the InterUSS
     Technical Steering Committee.
 
-A simple sequence diagram of USS A adding its record to the specific grid its operation is impacting. There are no other operations, so no need to contact other USSs (yet, more complicated sequences forthcoming).
```
Comment (PearGod) [inline README.md]: Why is this being deleted?

File: README.md [hunk 2]
```diff
@@ -76,23 +56,38 @@ When a USS wants to plan a flight, the “planning USS” performs the following
     of the world, industry provided OAuth solutions decided by the InterUSS
     Technical Steering Committee.
 
-A simple sequence diagram of USS A adding its record to the specific grid its operation is impacting. There are no other operations, so no need to contact other USSs (yet, more complicated sequences forthcoming). 
 ![Simple Sequence](assets/USS1.png)
```
Comment (PearGod) [inline README.md]: This graphic still refers to TCL4. Is there a link to the generation of the png that I can edit and update?

File: README.md [hunk 3]
```diff
@@ -76,23 +56,38 @@ When a USS wants to plan a flight, the “planning USS” performs the following
...
 
-In this sequence, USS A is already operating (and marked itself as wanting updates [announcement_level=True]. USS B and USS C are planning two different operations concurrently.
```
Comment (PearGod) [inline README.md]: Why is this deleted too?  Did you intend to delete USS1.png?

File: README.md [hunk 4]
```diff
@@ -76,23 +56,38 @@ When a USS wants to plan a flight, the “planning USS” performs the following
...
-multiple grids during an update. 
+multiple grids during an update. And finally, this
+[Sequence Diagram](assets/USS5.png) shows different examples of protection with
+concurrent updates.
 
 ## Directories of Interest:
+* `config/` has all of the configuration required to deploy a DSS instance to a kubernetes cluster. The README in that directory contains more information.
+* `pkg/` contains all of the source code for the DSS. See the README in that directory for more information.
+* `cmds/` contains entry points and docker files for the actual binaries (the `http-gateway` and `grpc-backend`)
+*
```
Comment (PearGod) [inline README.md]: Did you want to add assets to this list?  Otherwise, there's an extra bullet here.

File: README.md [hunk 5]
```diff
@@ -76,23 +56,38 @@ When a USS wants to plan a flight, the “planning USS” performs the following
...
+* `cmds/` contains entry points and docker files for the actual binaries (the `http-gateway` and `grpc-backend`)
+* 
+
+## Notes
+
+* Currently this branch only supports the Remote ID API's. 
+* The current implementation relies on CockroachDB for data storage and synchronization between DSS participants. It is recommended to read up on CockroachDB for performance characteristics and operational caveats. We list some of the caveats that we've run into below:
+
+### CockroachDB Notes
+* CockroachDB (CRDB) currently uses certificates to authenticate clients and node to node communication. All of Node certs, client certs, and even a CA cert are all generated through the cockroach cli. The CA certs must be custom generated since the Node certs are require to have CN=node, which no public CA will sign.
```
Comment (PearGod) [inline README.md]: Nit: We seem to be capitalizing "node" in "Node certs" but nowhere else, and we're not capitalizing client in "client certs".  Should the two instances instead be "node certs"?

File: README.md [hunk 6]
```diff
@@ -76,23 +56,38 @@ When a USS wants to plan a flight, the “planning USS” performs the following
...
+* CockroachDB nodes join the cluster virally. That is, each node keeps state on all the other nodes, and if a node connects to a new node, it will learn about the entire cluster through a gossip protocol.
+* Each CRDB node must be uniquely addressable and routable.
+  * Because of this, we expect each node to have it's own static IP and/or publicly resolvable hostname. In the future, we likely want to explore the use of Istio, which allows multi cluster private service discovery, layers on top it's own TLS so that we could run CRDB in insecure mode, and not worry about the certificate problems (Istio can use standard CA's), and would reduce latency by removing the extra hop of the CRDB per-node Loadbalancer we are currently using.
+* CRDB splits up data based on the locality string. Data replication strategy is a database variable. The CRDB nodes will traverse the key,values of the locality flag to determine how to divy up the replicas. This means there could be mroe than 7 participants of the DSS, with only 3 or 5 (or 7) replicas, and each participant would simply recieve a shard of a single replica.
```
Comment (PearGod) [inline README.md]: "mroe" -> "more"
